### PR TITLE
Add security auditor extra permissions

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR adds additional permissions that are required for security auditors to properly perform their job.

This PR also (belatedly) pulls in the latest updates from [`skeleton-tf-module`](https://github.com/cisagov/skeleton-tf-module), including `lineage` support so that it will be kept up-to-date automatically.

## 💭 Motivation and Context
The security auditors let us know that they were unable to view events in Cloudwatch logs, so I created a new policy allowing `logs:GetLogEvents` and `logs:FilterLogEvents` permissions.  That new `SecurityAuditExtras` policy was then added to the `security_auditors` group (in the Users account) and to the `SecurityAudit` role (in all of the other accounts).

## 🧪 Testing
I verified in Production that the expected policy was created in each account and that it was correctly added to the expected group/roles.

I also received confirmation from the security auditors themselves that they now had the necessary access to view log events in Cloudwatch.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ have been created and contain a ProvisionAccount role (ideally create via the
 
 None
 
+## Notes ##
+
+Running `pre-commit` requires running `terraform init` in every directory that
+contains Terraform code. In this repository, these are the main directory and
+every directory under `examples/`.
```
Comment (mcdonnnj) [inline README.md]: I believe it is only the main directory for this project.

**Review by mcdonnnj — APPROVED** [review]
Makes sense. Thanks for taking care of this and integrating Lineage + skeleton updates. Please see my one comment about a pulled in change.

**Review by jsf9k — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md]: Keen eyes there - thanks!  Fixed in https://github.com/cisagov/cool-users-non-admin/pull/2/commits/520e92d2364587b756238803d03a2b06fda34498.

**Pull request merged**